cmd/check/main.go

```diff
@@ -2,8 +2,14 @@ package main
 
 import (
 	"encoding/json"
+	"fmt"
```

Comment:
Looks like the prevailing style is to use `logrus` instead of `fmt`. This probably has more chance of being accepted upstream if you use calls like `logrus.Errorf()` in place of `fmt.Printf()`.  Other than that, this looks really good!

Reply:
Good spot, I'll change those